apps/wallet/src/ui/app/background-client/index.ts

```diff
@@ -562,6 +549,24 @@ export class BackgroundClient {
 		);
 	}
 
+	/**
+	 * Wallet wasn't storing the public key of ledger accounts, but we need it to send it to the dapps.
```

Comment:
this comment will be outdated in the not-to-distant future :)

Reply:
Not sure when we will be able to remove this. Even if all users update to the latest wallet there is a case where a ledger account was created using the old wallet and never connected the ledger device again - so we don't have the public key.